
[Executed at: Sun Mar 18 23:56:56 PDT 2018]

None
Traceback (most recent call last):
  File "/home/ccc_v1_c79523__48809/asn24778_45/asn24780_2/asnlib/autograde.py", line 14, in <module>
    score=runtest.main()
  File "/home/ccc_v1_c79523__48809/asn24778_45/asn24780_2/asnlib/runtestcases.py", line 66, in main
    stu_work=getattr(__import__(work_package, fromlist=[fun_name]), fun_name)
  File "./q4_work.py", line 28, in <module>
    print(determinant([100]))    
  File "./q4_work.py", line 14, in determinant
    col_n = len(matrix[0])
TypeError: object of type 'int' has no len()
Command exited with non-zero status 1
